### PR TITLE
flakytestfix

### PR DESCRIPTION
**Pull request by infanta-deriv**
card - https://app.clickup.com/t/20696747/DERC-2515
This PR has fix for flaky test QATEST-1366 Trade types ROW

### CODE REVIEW

**Review by meeyun-deriv — APPROVED** [review]
LGTM

**Review by abuzar-deriv — COMMENTED** [review]
Does this follow same formatting rules as e2e-deriv-app? If yes, need to remove semi-colons.

**Review by abuzar-deriv — APPROVED** [review]
Looks good

File: cypress/e2e/smoke/allHomeTradeType.cy.js [hunk 1]
```diff
@@ -18,22 +18,25 @@ const tradeTypeConfig = {
...
-    cy.get('@learnMoreLink').click()
-    cy.url().should('include', config.urls[index])
-    cy.go('back')
-  });
+  cy.findByRole('heading', { name: trade.name }).scrollIntoView().should('exist')
+  cy.findByText(trade.text).should('be.visible')
+  cy.findByRole('link', { name: `Learn more about ${trade.learnmore}` }).as('learnMoreLink')
+  cy.get('@learnMoreLink').trigger('mouseover')
+  cy.get('@learnMoreLink').click()
+  cy.url().should('include', config.urls[index])
```
Comment (bilalmajid-deriv) [inline cypress/e2e/smoke/allHomeTradeType.cy.js:34]: can you please add a check on content here as well?

**Review by bilalmajid-deriv — CHANGES_REQUESTED** [review]

**Review by bilalmajid-deriv — APPROVED** [review]

**Pull request merged**